packages/node_modules/@webex/plugin-meetings/sample/test/mediatest.js

```diff
@@ -0,0 +1,81 @@
+/* globals browser */
+import {expect} from 'chai';
+
+import testUsers from '../../../../@ciscospark/test-helper-test-users';
+
+describe('samples', () => {
+  describe('browser-call-with-screenshare', () => {
+    describe('normal dialing', () => {
+      let alice, bob;
+
+      const browserAlice = browser.select('browserAlice');
+      const browserBob = browser.select('browserBob');
+
+      before('create test users', () => testUsers.create({count: 2})
+        .then((users) => {
+          [bob, alice] = users;
+        }));
+
+      before('reload browser', () => {
+        browser.refresh();
+      });
+
+      it('loads the app', () => {
+        browser.url('/');
+        browser.waitForExist('#access-token', 1500);
+      });
+
+      it('connects bob\'s browser', () => {
+        expect(browserBob.getTitle()).to.equal('Sample: Meetings Kitchen Sink');
+        browserBob.setValue('#access-token', bob.token.access_token);
+        browserBob.pause(10000);
+        browserBob.click('#connect');
+
+        browserBob.waitUntil(() => {
+          // Test runs too quickly sometimes and tries to click before call is fully established
+          if (browserBob.getText('#connection-status') !== 'connected') {
+            return false;
+          }
+          return true;
+        }, 10000, 'Timed-out waiting for login');
+      });
+
+      it('connects alice\'s browser', () => {
+        expect(browserAlice.getTitle()).to.equal('Sample: Meetings Kitchen Sink');
+        browserAlice.setValue('#access-token', alice.token.access_token);
+        browserAlice.pause(10000);
+        browserAlice.click('#connect');
+        browserAlice.waitUntil(() => {
+          // Test runs too quickly sometimes and tries to click before call is fully established
+          if (browserAlice.getText('#connection-status') !== 'connected') {
+            return false;
+          }
+          return true;
+        }, 10000, 'Timed-out waiting for login');
+      });
+
+      it('places call from bob to alice', () => {
```

Comment:
It looks like each of these tests are dependent on the previous ones.

Reply:
Ya @mattnorris  Kept it this way as its sample automation . Unit and integration tests are single it . The previous one are dependent but should not run test if previous one are failing . We can change it to the promise chain and function names .  But its still hard to debug and understand when someone new looks at it .  Still like nested describe compared to one it section @chrisadubois  @andrewholsted  thoughts

Reply:
I think the phone plugin uses the same format 
https://github.com/webex/spark-js-sdk/blob/master/packages/node_modules/samples/browser-single-party-call/test/wdio/spec/dial-and-reject.js

Reply:
I still don't like that because the tests should be independent. I don't think it should have been done in `phone` this way. @arun3528 what is an example of the nested describes?

Reply:
@arun3528 if we understand why we are breaking from conventions as a team, that's okay with me. I just want consistency amongst the team.